general/calendars/outlook/outlook_calendar.rst

```diff
@@ -0,0 +1,57 @@
+=====================================
+Synchronization with Outlook Calendar
+=====================================
+
+Life is a lot easier when you are able to manage all your calendars in one single place. Therefore,
+now Odoo also allows you to synchronize your Outlook Calendar with the Calendar application.
+
+To start, `Register your application with the Microsoft identity platform
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_ making
+sure to **follow all the steps described in the guidelines**.
+
+.. important::
+   For the **Redirect URI restrictions**, copy your Odoo server URI followed by
+   ``/microsoft_account/authentication``. Example:
```

Comment:
I think it is a good idea to link to Microsoft's help, although it could be a bit more detailed here as well.
For example, to create a **web** based app on azure, and of course the URI addition for the app to work.